src/lang-promql/complete/index.ts

```diff
@@ -32,19 +34,20 @@ export interface CompleteStrategy {
 
 // CompleteConfiguration should be used to customize the autocompletion.
 export interface CompleteConfiguration {
-  // Provide these settings when not using a custom PrometheusClient.
-  url?: string;
-  lookbackInterval?: number;
-  httpErrorHandler?: (error: any) => void;
-  fetchFn?: FetchFn;
-
+  remote: {
```

Comment:
Shouldn't the whole `remote` property also be marked optional?

Reply:
ah yeah that is totally true :)